src/Blubrry/REST/Resource/Social.php

```diff
@@ -79,6 +177,44 @@ public function postSocial($program_keyword, $body) {
             }
         }
 
+        $type=$body['social-type'];
+        $twitter=array('content');
+
+        $facebook=array(
+            "title-$body['social-id']",
+            "description-$body['social-id']",
+            "destination-$body['social-id']",
+        );
+
+        $youtube=array(
+            "title-$body['social-id']",
+            "description-$body['social-id']",
+        );
+
+        if ($type == "twitter") {
```

Comment:
@lucashillebrandt I would recommend to use Yoda conditions instead. Please read here: https://knowthecode.io/yoda-conditions-yoda-not-yoda